cclib/method/volume.py

```diff
@@ -315,12 +350,13 @@ def electrondensity(ccdata, volume, mocoeffslist):
     """
 
     if len(mocoeffslist) == 2:
-        return electrondensity_spin(ccdata, volume, [mocoeffslist[0]]) + electrondensity_spin(ccdata, volume, [mocoeffslist[1]])
+        return electrondensity_spin(ccdata, volume, [mocoeffslist[0]]) + electrondensity_spin(
+            ccdata, volume, [mocoeffslist[1]]
+        )
     else:
         edens = electrondensity_spin(ccdata, volume, [mocoeffslist[0]])
         edens.data *= 2
         return edens
-    
-del find_package
 
-    
\ No newline at end of file
+
+del find_package

```

Comment:
What is the purpose of this this `del`?

Reply:
This looks like something from the existing code. I'll see how the tests go, but I don't think it serves any purpose here. Thanks for letting me know about this!